docs: Update screenshot-and-gif-software document.

adarshPatel509 opened the pull request.

timabbott commented:
@adarshPatel509 what's your thinking about the ordering of these additions in relation to the existing tools?  I'm in particular curious about OSX.

adarshPatel509 commented:
@timabbott ,I added QuickTime in top of the list, because its free software and its already installed in most OSX distributions and other softwares need to be installed additionally.
and for screenshots I added "Cmd-Shift-5" option in the last ,because its not supported by all OSX distributions (Mojave only)
